src/assets/labContent.js

```diff
@@ -44,21 +40,19 @@ export default [
       subheader: "How do the appearance of ads differ across individual accounts?",
       bodyText: [
         {
-          body: `Computer programs categorize users based on their demographical characteristics in order to optimize efficiency in many things, including advertising. By incorporating these categorizations into advertising algorithms, advertisers are able to maximize the relevancy of their ads and thus maximize their profit.`
+          body: <text>Computer programs <mark style={{backgroundColor: '#ffedbd', borderRadius: '7px', padding: '0 5px 5px 5px'}}>categorize</mark> users based on their demographical characteristics in order to optimize efficiency in many things, including advertising. Through such categorizations, advertisers are able to <u>maximize the relevancy of their ads and thus maximize their profit.</u></text>
         },
         {
-          body: `How do the decisions made by such algorithms impact the type of ads that we receive on social media?`,
-          bold: true
+          body: <text>Let's say you decided to create a profile on a social media app that displays advertisements "catered" to your preferences. <b>How does changing your profile affect the ads that you receive?</b></text>
         },
         {
-          body: `Let's say you decided to create a profile on a social media app that displays advertisements "catered" to your preferences. How does changing your profile affect the ads that you receive?`
+          body:<text style={{color:'gray'}}>Why are the options for gender and race binary, you ask? That's because this activity is based on the data from a <a href='https://arxiv.org/pdf/1904.02095.pdf' target="_blank" rel="noopener noreferrer">research study</a> that only considers these four criteria in order to simplify their experimental process. Teach LA recognizes that race and gender are <i>not</i> binary, and that it is crucial for algorithms to be inclusive of all identities! </text>
```

Comment:
I like the idea of having some color contrast for further insight on why our ad section is set up to be binary, but I find it hard to read since it is too similar to the white background. I think making it a bit darker would help readability! (Maybe try running a fully loaded site through an accessibility checker like I said earlier?)